app/views/static/home.html.erb

```diff
@@ -1,11 +1,187 @@
-Erudition
-</br>
-</br>
-<% if !user_signed_in? %>
-  <%= link_to 'Sign up', new_user_registration_path %>
-  </br>
-  <%= link_to 'Log in', new_user_session_path %>
-<% end %>
-<% if user_signed_in? %>
-  <%= link_to 'Log out', destroy_user_session_path, method: :delete %>
-<% end %>
+<div class="page page-home">
+
+  <nav class="navbar navbar-default">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="/">Erudition</a>
+      </div>
+      <div class="navbar-header navbar-right">
+        <% if !user_signed_in? %>
+          <%= link_to 'Log in', new_user_session_path, class: 'btn btn-default navbar-btn nav-pills' %>
+        <% end %>
+        <% if user_signed_in? %>
+          <p class="navbar-text">Signed in as email@email.com</p>
+          <%= link_to 'Log out', destroy_user_session_path, method: :delete, class: 'btn btn-default navbar-btn' %>
+        <% end %>
+      </div>
+    </div>
+  </nav>
```

Comment:
That makes sense. The reality is that we'll end up shipping V1 before arranging any AB testing. I also imagine that the `nav` and `footer` will be on every page except _learn mode_ pages? If that's the case we could define a new layout for _learn mode_ that does not include the `nav` and `footer`.